substrate/frame/src/lib.rs

```diff
@@ -505,7 +508,7 @@ pub mod runtime {
 	#[cfg(feature = "std")]
 	pub mod testing_prelude {
 		pub use sp_core::storage::Storage;
-		pub use sp_runtime::BuildStorage;
+		pub use sp_runtime::{BuildStorage, DispatchError::Unavailable};
```

Comment:
I am not sure `Unavailable` fits into prelude, `DispatchError` should be enough I think.

Reply:
Yeah, I do think it is a bit too much to import enum `variants` in any prelude -- the Enum itself makes a lot of sense. `DispatchError` is super popular, and 100% should be in prelude. 

If a code wants to use `Unavailable`, they would just use `DispatchError::Unavailable` and that is fine. 

Please add this (among other guidelines) to `substrate/frame/src/lib.rs`, in the top of the file, there is some high level doc, and a section exactly explaining the rules of the umbrella crate.